cms/static/js/views/list_item.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * A generic view to represent an editable item in a list.  The item
+ * has a edit view and a details view.
+ *
+ * Children Must implement:
+ * - itemDisplayName (string): Display name for the list item.
+ * - createEditView (function): Render and append the edit view to the
+ *   DOM.
+ * - createDetailsView (function): Render and append the details view
+ *   to the DOM.
+ */
+define([
+    'js/views/baseview', 'jquery', "gettext", "js/views/utils/view_utils"
+], function(
+    BaseView, $, gettext, ViewUtils
+) {
+    'use strict';
+
+    var ListItem = BaseView.extend({
+        canDelete: false,
+
+        initialize: function() {
+            this.listenTo(this.model, 'change:editing', this.render);
+            this.listenTo(this.model, 'remove', this.remove);
+        },
+
+        className: function () {
+            var index = this.model.collection.indexOf(this.model);
+
+            return [
+                'group-configuration',
```

Comment:
I think it makes sense to factor out 'group-configuration' into the two classes, and have something like the following in list_item.js:

```javascript
return [
  this.baseClassName,
  this.baseClassName + '-list-item',
  this.baseClassName + '-list-item-' + index
]
```

But as it is, both classes share the same CSS class names. I'm going to wait till we hear back from @frrrances to change anything here.